tests/common/devices/sonic_asic.py

```diff
@@ -325,6 +326,9 @@ def bgp_drop_rule(self, ip_version, state="present"):
 
         logger.debug(output)
 
+    def get_rpc_port_ssh_tunnel(self):
```

Comment:
Will the rpc ports have conflict for multi-dut?

Reply:
The tunnel is created based on combination of DUT's mgmt_ip and port that is unique. So, no conflict for multi-dut.